apps/sms/views/conversation/js/conversation.js

```diff
@@ -721,6 +723,48 @@ var ConversationView = {
     });
   },
 
+  handleComposer: function thui_handleComposer(composer, recipients) {
+
+    function generateMmsAttachmentsForComposer(slides, content) {
+      if (typeof content === 'string') {
+        slides.push({
+          content: new Blob([content], { type: 'text/plain' }),
+        });
+      } else {
+        slides.push({
+          content: content.blob,
+          location: content.name
+        });
+      }
+      return slides;
+    }
+
+    function addRecipientsToConversation(data) {
+      if (data.type !== '') {
```

Comment:
That's the way I came up with to know if I found a contact in the user DB or not. Not very satisfying imo. Do you have better?